Mathlib/RingTheory/PrincipalIdealDomain.lean

```diff
@@ -489,6 +489,14 @@ theorem exists_associated_pow_of_mul_eq_pow' {a b c : R} (hab : IsCoprime a b) {
   exact exists_associated_pow_of_mul_eq_pow ((gcd_isUnit_iff _ _).mpr hab) h
 #align exists_associated_pow_of_mul_eq_pow' exists_associated_pow_of_mul_eq_pow'
 
+theorem exists_associated_pow_of_associated_mul_pow {a b c : R} (hab : IsCoprime a b) {k : ℕ}
```

Comment:
shuold it be `...of_associated_mul_pow` or `...of_associated_pow_mul`? In general, what is the naming convention for associated things?

Reply:
You're right, I switched the order at some point and didn't change the name.